packages/sdk/typescript/human-protocol-sdk/src/kvstore.ts

```diff
@@ -277,37 +277,120 @@ export class KVStoreClient extends BaseEthersClient {
+ * ### yarn
+ * ```bash
+ * yarn install @human-protocol/sdk
+ * ```
+ *
+ * ## Code example
+ *
+ * ### Signer
+ *
+ * **Using private key (backend)**
+ *
+ * ```ts
+ * import { ChainId, KVStoreUtils } from '@human-protocol/sdk';
+ *
+ * const KVStoreAddresses = new KVStoreUtils.getKVStoreData({
+ *   ChainId.POLYGON_AMOY,
+ *   "0x1234567890123456789012345678901234567890",
+ * );
+ * ```
+ */
+export class KVStoreUtils {
+  /**
+   * This function returns the KVStore data for a given address.
+   *
+   * @param {ChainId} chainId Network in which the KVStore is deployed
+   * @param {string} address Address of the KVStore
+   * @returns {Promise<IKVStore[]>} KVStore data
+   *
+   * **Code example**
+   *
+   * ```ts
+   * import { ChainId, KVStoreUtils } from '@human-protocol/sdk';
+   *
+   * const kvStoreData = await KVStoreUtils.getKVStoreData(ChainId.POLYGON_AMOY, "0x1234567890123456789012345678901234567890");
+   * console.log(kvStoreData);
+   * ```
+   */
+  public static async getKVStoreData(
+    chainId: ChainId,
+    address: string
+  ): Promise<IKVStore[]> {
+    const networkData = NETWORKS[chainId];
+
+    if (!networkData) {
+      throw ErrorUnsupportedChainID;
+    }
+
+    if (address && !ethers.isAddress(address)) {
+      throw ErrorInvalidAddress;
+    }
+
+    const { kvstores } = await gqlFetch<{ kvstores: KVStoreData[] }>(
+      getSubgraphUrl(networkData),
+      GET_KVSTORE_BY_ADDRESS_QUERY(),
+      { address: address.toLowerCase() }
+    );
+
+    const kvStoreData = kvstores.map((item) => ({
+      key: item.key,
+      value: item.value,
+    }));
+
+    return kvStoreData || [];
+  }
 
   /**
    * Gets the value of a key-value pair in the contract.
    *
+   * @param {KVStore} contract The KVStore contract instance.
    * @param {string} address Address from which to get the key value.
    * @param {string} key Key to obtain the value.
-   * @returns {string} Value of the key.
-   *
+   * @returns {Promise<string>} Value of the key.
    *
    * **Code example**
    *
-   * > Need to have available stake.
-   *
    * ```ts
+   * import { KVStore__factory, KVStoreUtils } from '@human-protocol/sdk';
    * import { providers } from 'ethers';
-   * import { KVStoreClient } from '@human-protocol/sdk';
    *
    * const rpcUrl = 'YOUR_RPC_URL';
+   * const address = '0x1234567890123456789012345678901234567890';
+   * const key = 'Role';
    *
    * const provider = new providers.JsonRpcProvider(rpcUrl);
-   * const kvstoreClient = await KVStoreClient.build(provider);
+   * const kvstoreContract = KVStore__factory.connect(NETWORKS[ChainId.POLYGON_AMOY]?.kvstoreAddress, provider);
    *
-   * const value = await kvstoreClient.get('0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266', 'Role');
+   * const value = await KVStoreUtils.get(kvstoreContract, address, key);
+   * console.log(value);
    * ```
    */
-  public async get(address: string, key: string): Promise<string> {
+  public static async get(
+    contract: KVStore,
```

Comment:
Instead of passing the contract here, I would pass the rpc url and use the address of the contract from constants based on chainID.
The same for other functions that you are passing the contract